src/java.desktop/share/classes/javax/swing/undo/UndoableEditSupport.java

```diff
@@ -148,7 +147,7 @@ public int getUpdateLevel() {
     }
 
     /**
-     *
+     * Start UndoableEdit.
```

Comment:
What does it mean? Is it clear enough for the class which "managing <code>UndoableEdit</code> listeners"?

Reply:
It seems it other places like CompoundEdit class, "assemble little UndoableEdits " statement is used so it seems ok to use UndoableEdit here. At least it's better than (to me)
```
"DEADLOCK WARNING: Calling this method may call
     * <code>undoableEditHappened</code> in all listeners."
```
   
 used in endUpdate...

Reply:
Can we update the docs for those methods with a more meaningful description? In a separate issue.